src/fmu/dataio/_objectdata_provider.py

```diff
@@ -560,11 +591,12 @@ def _derive_timedata(self):
             timedata = self._derive_timedata_newformat()
         return timedata
 
-    def _derive_timedata_legacy(self):
+    def _derive_timedata_legacy(self) -> Dict[str, Any]:
```

Comment:
Another few here that could be lower cased `dict`s?

Reply:
Leftovers from one of mypy battles.